Provider/src/main/java/com/spotify/confidence/client/ConfidenceRemoteClient.kt

```diff
@@ -45,6 +45,7 @@ class ConfidenceRemoteClient : ConfidenceClient {
             "application/json"
         )
         baseUrl = when (region) {
+            ConfidenceRegion.GLOBAL -> "https://resolver.confidence.dev/"
```

Comment:
```suggestion
            ConfidenceRegion.GLOBAL -> "https://resolver.confidence.dev"
```
please drop the last `/` to be consistent.